### DISCUSSION TITLE
mod tools

### DISCUSSION THREAD

**Discussion by Tiritibambix** [Docile-Alligator/Infinity-For-Reddit#948] [Ideas]
It would be great to have access to mod tools such as lock, spam, remove, approve, distinguish and sticky :)

**Comment by ghost** [comment]
Plus one for this

**Comment by Alvie** [comment]
Another plus one for this, I think this is the only thing keeping it from being the best third party Reddit app.

Hopefully it's not too difficult to implement such a thing.

Maybe also in the navigation drawer, a moderation section as well where you can view mod logs, mod mail (and filter by subreddits that you moderate for). Perhaps this would be easier to implement before adding the post moderation tools (lock/sticky/distinguish/approve/etc)

**Comment by chesamo** [comment]
Adding another vote of support-- I'm trying to transition off Slide (since it's not actively developed anymore) and this is one of the roadblocks.

**Comment by Pentaphon** [comment]
This is the only reason Infinity can't be my only Reddit client.

**Comment by pe1uca** [comment]
@Docile-Alligator is there any update or plan on this?  
I'm working on some personal scripts to mod from mobile since Infinity doesn't have this yet, so I might as well put the effort into this rather than scripts which will be discarded.
